libexec/rtld-elf64cb-c18n/Makefile

```diff
@@ -0,0 +1,11 @@
+NEED_COMPAT=	64CB
```

Comment:
Just bundle all this (and libc/libthr) with the commit to change the assembly, you just want a single "benchmark ABI c18n userspace"-like commit